chassis/core/src/main/java/com/griddynamics/jagger/user/test/configurations/limits/auxiliary/JMetricName.java

```diff
@@ -0,0 +1,214 @@
+package com.griddynamics.jagger.user.test.configurations.limits.auxiliary;
+
+import com.griddynamics.jagger.util.StandardMetricsNamesUtil;
+
+/**
+ * Enum for standard metrics.
+ */
+public enum JMetricName {
+
+    SUCCESS_RATE_OK,
+    SUCCESS_RATE_FAILS,
+    THROUGHPUT,
+    VIRTUAL_USERS,
+    STD_DEV_LATENCY,
+    DURATION,
+    AVG_LATENCY,
+    ITERATION_SAMPLES,
+    MON_CPULA_1,
+    MON_CPULA_5,
+    MON_CPULA_15,
+    MON_GC_MINOR_TIME,
+    MON_GC_MAJOR_TIME,
+    MON_GC_MINOR_UNIT,
+    MON_GC_MAJOR_UNIT,
+    MON_MEM_RAM,
+    MON_MEM_TOTAL,
+    MON_MEM_USED,
+    MON_MEM_ACTUAL_USED,
+    MON_MEM_FREE_PRCNT,
+    MON_MEM_ACTUAL_FREE,
+    MON_MEM_FREE,
+    MON_TCP_EST,
+    MON_TCP_LISTEN,
+    MON_SYNC_RECEIVED,
+    MON_INBOUND_TOTAL,
+    MON_OUTBOUND_TOTAL,
+    MON_DISK_READ_BYTES,
+    MON_DISK_WRITE_BYTES,
+    MON_DISK_SERVICE_TIME,
+    MON_DISK_QUEUE_SIZE_TOTAL,
+    MON_CPU_USER,
+    MON_CPU_SYS_PRCNT,
+    MON_CPU_IDLE_PRCNT,
+    MON_CPU_WAIT,
+    MON_CPU_COMBINED,
+    MON_HEAP_INIT,
+    MON_HEAP_USED,
+    MON_HEAP_COMMITTED,
+    MON_HEAP_MAX,
+    MON_NONHEAP_INIT,
+    MON_NONHEAP_USED,
+    MON_NONHEAP_COMMITTED,
+    MON_NONHEAP_MAX,
+    MON_THREAD_COUNT,
+    MON_THREAD_PEAK_COUNT,
+    MON_FILE_DESCRIPTORS;
+
+
+    /**
+     * Convert {@link JMetricName} to String.
+     *
+     * @return String value of current metric name.
+     */
+    public String transformToString() {
+        String name = null;
+        switch (this) {
+            case SUCCESS_RATE_OK:
+                name = StandardMetricsNamesUtil.SUCCESS_RATE_OK_ID;
+                break;
```

Comment:
you can return the value right away.
also, you can use a Map with JMetricName as key and corresponding values. and here just get that value by a key